app/retail/templates/leaderboard.html

```diff
@@ -17,117 +17,133 @@
       {% include 'shared/nav.html' %}
     </div>
     <div class="row no-gutter">
-      <div class="col body">
+      <div class="col body gitcoin-background">
         <div class="container-fluid" id="leaderboard">
-          <div class="offset-md-1 col-sm-12 col-md-10">
-            <div class="row mt-0 mb-2 pt-1 podium">
+
+          <!-- PODIUM -->
+
+          <div class="row offset-md-1 col-sm-12 col-md-10 mt-0 mb-2 pt-1 podium text-center justify-content-center" style="background-color: #0d023b">
               {% for podium_item in podium_items %}
-                <div class="col-4 winner">
+                <div class="col-2 winner mt-5">
```

Comment:
I saw there are 5 leaders aligned to the leader box edges so I think will be better to use d-flex and others bootstrap flex classes instead of the grid, in that way will be possible to get the pictures aligned to the edges.

Reply:
here are some helpful docs https://getbootstrap.com/docs/4.3/utilities/flex/#justify-content

Reply:
![image](https://user-images.githubusercontent.com/1358093/54309822-b3d22d80-45af-11e9-89e0-99fce5b46f5e.png)

Reply:
All flex options are breaking the container:
![](https://i.imgur.com/ijoksSw.png)

Reply:
This is fixed